doc/api/buffer.md

```diff
@@ -279,6 +279,101 @@ for (const b of buf) {
 Additionally, the [`buf.values()`][], [`buf.keys()`][], and
 [`buf.entries()`][] methods can be used to create iterators.
 
+## Class: `Blob`
+<!-- YAML
+added: REPLACEME
+-->
+
+A `Blob` encapsulates immutable, raw data.
```

Comment:
Add a reference to MDN/the spec here?

Reply:
I'd prefer to wait until we're sure we're compliant